software-containers/README.md

```diff
@@ -88,7 +36,7 @@ IP_ADDRESS=`hostname --ip-address`
 ```
 ##### Use The UI
 If you have previously created a desktop session, you can use the UI of that session to access your container. There is an 'Applications' tab on the top left hand corner. Select 'Applications->CANFAR->CASA' and then the version under development. An xterm for the selected CASA version will pop up after some time.
```

Comment:
You can repurpose this section to indicate that "you will need to run the container in a graphical environment, such as arcade, to test the visual components of the CASA containers".  But no other details about Arcade please.  Those things will likely all change and we don't want to have to come back and update this.

Reply:
Okay